codesigning/enqueue-current-installer.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
+#
+# This script determines the filename of the binary that is built on the
+# current operating system and enqueues it for code signing.
+#
+# NOTE: this script must be run from the directory containing this script.
+
+version=$(python -m setuptools_scm)
+url_base=$(make -C .. --no-print-directory print-DIST_URL_BASE | awk ' { print $3 } ')
+platform=$(python -c "import platform;p=platform.system().lower();print(p if p != 'windows' else 'win32')")
+
+if [ "$platform" = "win32" ]; then
```

Comment:
Would it be worth scripting this in python? Since there's already some python commands in here, and the other codesigning scripts are in python.

Reply:
Yes!  Absolutely, both because several components of this are python anyways and also because python will allow us to run this script from any CWD, as long as we know the location of the script (which python should always know).  Patched in 99e294921